### PR TITLE
feat(website): add card documentation

### PR DESCRIPTION
**Pull request by bbogdanov**
Adding `cds-card` documentation in the website

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] If applicable, have a visual design approval

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [x] clarity.design website / infrastructure changes
- [ ] Other... Please describe:

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by mathisscott — APPROVED** [review]
Make sure @colinreedmiller or @yuxin-ux get to check this out.

File: apps/website/.vuepress/theme/global-components/CdsDemos.vue [hunk 1]
```diff
@@ -1,19 +1,23 @@
 <template>
   <div v-if="component.stories">
-    <!--    <h2 cds-layout="m-b:xl">{{ componentName | uppercase }} Code</h2>-->
-    <section cds-layout="m-b:xl" v-for="story in component.stories">
-      <h2 cds-layout="m-b:md">{{ story.storyName | uppercase }}</h2>
+    <!-- <h2 cds-layout="m-b:xl">{{ componentName | uppercase }} Code</h2> -->
+    <section cds-layout="m-b:xl" v-for="(story, key) in component.stories" :key="key">
+      <h2 cds-layout="m-b:md">{{ story.storyName | uppercaseSplit }}</h2>
```
Comment (hippee-lee) [inline apps/website/.vuepress/theme/global-components/CdsDemos.vue:5]: 👍🏼

File: apps/website/core-components/card/README.md [hunk 2]
```diff
@@ -0,0 +1,251 @@
+---
+title: Overview
+toc: true
+---
+
+Cards are used to apply a container around a related grouping of information.
```
Comment (hippee-lee) [inline apps/website/core-components/card/README.md]: markdown content should be wrapped inside of the appropriate blocks to apply the correct core spacing tokens to the generated content. 
E.g:

``` markdown
::: component-summary
Summary of the component goes here.
:::
```

The same goes for the various level headings and associated description. Any one of the existing core-component md files should be following this pattern.

**Review by hippee-lee — CHANGES_REQUESTED** [review]

**Review by hippee-lee — COMMENTED** [review]
lgtm 👍🏼

**Review by hippee-lee — APPROVED** [review]

**Comment by bbogdanov** [comment]
#5785

**Comment by bbogdanov** [comment]
#5563

**Pull request merged**